site/zenodo_rdm/legacy/vocabularies/licenses.py

```diff
@@ -0,0 +1,167 @@
+    "Frameworx-1.0": "frameworx",
+    "FAL-1.3": "fal",
+    "GFDL-1.3-no-cover-texts-no-invariant-sections": "gfdl",
+    "GPL-2.0": "gpl-2.0",
+    "GPL-3.0": "gpl-3.0",
+    "LGPL-2.1": "lgpl-2.1",
+    "LGPL-3.0": "lgpl-3.0",
+    "geogratis": "geogratis",
+    "hesa-withrights": "hesa-withrights",
+    "IPL-1.0": "ibmpl",
+    "IPA": "ipafont",
+    "ISC": "isc-license",
+    "Intel": "intel-osl",
+    "notspecified": "notspecified",
+    "localauth-withrights": "localauth-withrights",
+    "LPL-1.02": "lucent1.02",
+    "MIT": "mit-license",
+    "mitre": "mitre",
+    "met-office-cp": "met-office-cp",
+    "MS-PL": "ms-pl",
+    "MS-RL": "ms-rl",
+    "MirOS": "miros",
+    "Motosoto": "motosoto",
+    "MPL-1.0": "mozilla",
+    "MPL-1.1": "mozilla1.1",
+    "Multics": "multics",
+    "NASA-1.3": "nasa1.3",
+    "NTP": "ntp-license",
+    "Naumen": "naumen",
+    "NGPL": "nethack",
+    "Nokia": "nokia",
+    "NPOSL-3.0": "nosl3.0",
+    "OCLC-2.0": "oclc2",
+    "ODC-BY-1.0": "odc-by",
+    "ODbL-1.0": "odc-odbl",
+    "ODC-PDDL-1.0": "odc-pddl",
+    "OFL-1.1": "openfont",
+    "OGTSL": "opengroup",
+    "OSL-3.0": "osl-3.0",
+    "other-at": "other-at",
+    "other-nc": "other-nc",
+    "other-closed": "other-closed",
+    "other-open": "other-open",
+    "other-pd": "other-pd",
+    "PHP-3.0": "php",
+    "Python-2.0": "pythonsoftfoundation",
+    "CNRI-Python": "pythonpl",
+    "QPL-1.0": "qtpl",
+    "RPSL-1.0": "real",
+    "RPL-1.5": "rpl1.5",
+    "RSCPL": "ricohpl",
+    "SimPL-2.0": "simpl-2.0",
+    "Sleepycat": "sleepycat",
+    "dli-model-use": "dli-model-use",
+    "SISSL": "sun-issl",
+    "SPL-1.0": "sunpublic",
+    "Watcom-1.0": "sybase",
+    "Talis": "tcl",
+    "PostgreSQL": "postgresql",
+    "ukclickusepsi": "ukclickusepsi",
+    "ukcrown-withrights": "ukcrown-withrights",
+    "ukcrown": "ukcrown",
+    "OGL-UK-1.0": "uk-ogl",
+    "ukpsi": "ukpsi",
+    "NCSA": "uoi-ncsa",
+    "VSL-1.0": "vovidapl",
+    "W3C": "w3c",
+    "Xnet": "xnet",
+    "ZPL-2.0": "zpl",
+    "WXwindows": "wxwindows",
+    "Zlib": "zlib-license",
+    "CC-BY-NC-4.0": "cc-by-nc-4.0",
+}
+
+INVERSE_LEGACY_ALIASES = {v: k for k, v in LEGACY_ALIASES.items()}
+
+# Licenses mapping (Legacy->RDM)
+LEGACY_TO_RDM_MAP = _load_legacy_to_rdm_map()
+
+# Inverse mapping (RDM->Legacy)
+RDM_TO_LEGACY_MAP = {v: k for k, v in LEGACY_TO_RDM_MAP.items()}
+
+
+def legacy_to_rdm(license):
+    """Returns an RDM license given a zenodo legacy license."""
+    alias = LEGACY_ALIASES.get(license)
+
+    license = alias or license
+
+    rdm_license = LEGACY_TO_RDM_MAP.get(license)
+
+    # Return matched license or None
+    return rdm_license
+
+
+def rdm_to_legacy(right):
+    """Returns a zenodo legacy license given an RDM license."""
+    legacy_license = RDM_TO_LEGACY_MAP.get(right)
+
+    alias = INVERSE_LEGACY_ALIASES.get(legacy_license)
+    legacy_license = alias or legacy_license
+
+    # Return matched license or None
+    return legacy_license
```

Comment:
```suggestion
    return legacy_license or right
```

minor: if the legacy license was not found, it's ok to just return the rdm license